engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/UniversalSourceJsonFunctionalTest.scala

```diff
@@ -77,9 +98,10 @@ class UniversalSourceJsonFunctionalTest extends AnyFunSuite with Matchers with S
   test("should catch runtime errors") {
```

Comment:
can we have also assertions whether error occurs in sink or source?

Reply:
Changed.